iiblib/iib_build_details_model.py

```diff
@@ -308,11 +306,9 @@ class RmModel(IIBBuildDetailsModel):
         "removed_operators",
         "organization",
         "distribution_scope",
+        "deprecation_list"
     ]
 
-    _optional_attrs = IIBBuildDetailsModel._optional_attrs
-    _optional_attrs.update({"deprecation_list": lambda: list()})
```

Comment:
Please update the comment that you introduced this. Then you do not have to remove it at all.